package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
@larssn if you want to include SCSS with your module, you need to copy the files from `src` to `dist`. I'm doing that in a very inefficient way [here](https://github.com/zyra/ionic2-super-tabs/blob/master/package.json#L10). Perhaps I should add a gulp task or find a CLI utility that can copy the files properly.